<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>536</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="positions_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="back_button">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="play_button">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_button">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="saveBox">
          <property name="text">
           <string>save history</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="openFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openFolder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFolder_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>play_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>play_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pause_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>back_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFolder_button()</slot>
  <slot>play_button()</slot>
  <slot>pause_button()</slot>
  <slot>back_button()</slot>
  <slot>next_button()</slot>
  <slot>volume_slider()</slot>
  <slot>pozitions_slider()</slot>
 </slots>
</ui>
